source/_integrations/moat.markdown

```diff
@@ -0,0 +1,26 @@
+---
+title: Moat
+description: Instructions on how to integrate Moat devices into Home Assistant.
+ha_category:
+  - Sensor
+ha_zeroconf: true
```

Comment:
Should this be a front matter item for bluetooth discovery?

Reply:
```suggestion
ha_bluetooth: true
```